fastddsspy_tool/test/application/CMakeLists.txt

```diff
@@ -0,0 +1,200 @@
+# Copyright 2023 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###########################################################################
+# Create tests                                                            #
+###########################################################################
+
+find_package(PythonInterp 3 REQUIRED)
+
+# Name of files to test
+set(TEST_LIST
+    one_shot__config
```

Comment:
I'd modify these list for an auto-find cmake function that can find every file in this directory (I would also all tests to dir `test_cases` so they are not mixed with other files as `test.py`). This way, adding new tests is easier and reviewer do not have to check that all tests are included
Check https://github.com/eProsima/dev-utils/blob/acb7eca9c2119ec01265a5b736effe58e22544f9/cmake_utils/cmake/compilation/compile_library.cmake#L36